doc/tech/usecases.md

```diff
@@ -177,13 +177,25 @@ CNTT believes that Edge computing is unique in terms of infrastructure requireme
 
         Note (3) - The potential use cases that may consume services from Edge is the Network Model (V2N) and potentially the V2I (According on how the Infrastructure will be mapped to an Edge level)
 
-- **Use Case #4**
+- **Use Case #4 – vRAN**
 
   - **Business Objectives**
-
+      vRAN is one of the trending technologies of RAN deployment that fits for all Radio Access Technologies. vRAN helps to provide coverage for rural & uncovered areas with a compelling CAPEX reduction compared to Traditional and legacy RAN deployments. This coverage can be extended to all area types with 5G greenfield deployment as a typical example. 
+  
   - **Targeted Segments**
+  
+      - Private 5G Customers (vRAN Can be part of the Non-Public Network, NPN)
+      - B2B Customers & MVNOs (vRAN Can be part of an E2E Network Slicing)
+      - B2C (Mobile Consumers Segment).
 
+  
   - **Architecture**
+      
+      <img width="1068" alt="Screenshot 2020-05-11 at 5 51 34 PM" src="https://user-images.githubusercontent.com/25845305/81582240-15f31200-93b0-11ea-993e-a89f820a5d97.png">
+      
+        Note (1) – There are multiple deployment models for Centralized Unit (CU) & Distributed Unit (DU). This use case covers the placement case of having the DU & CU collocated & deployed on Telco Edge.
```

Comment:
For virtualised RAN (vRAN) there are multiple deployment options based on the latency requirements. For example, depending upon the latency requirements the CU could be deployed to, say, a telco central office site.

Reply:
Yes. There are other multiple deployment models but I am highlighting in the note that this use case covers the collocation of CU/DU in Telco Edge. NGMN has a nice digram showing all options.

Reply:
This comment will also be resolved by including appropriate references.

Reply:
@pgoyal01 I added the reference.
However, I believe maybe it is better to group all references in a dedicated section. Because there are a lot of references when it comes to vRAN, 5G NR, & Deployment Options.

Reply:
Hi there, I agree with Karim, the collocation of DU/CU is also the first scenario in O-RAN we have chosen to investigate in detail, please see "O-RAN Cloud Platform Reference Design for Deployment Scenario B" specification here https://www.o-ran.org/specifications. Thanks. BR.